contracts/grpc-contracts/src/main/proto/lifeos/finance/v1/finance_summary.proto

```
@@ -0,0 +1,28 @@
+import "lifeos/common/v1/request_metadata.proto";
+
+option java_multiple_files = true;
+option java_outer_classname = "FinanceSummaryProto";
+option java_package = "com.lifeos.grpc.finance.v1";
+
+service FinanceSummaryService {
+  rpc GetPeriodSummary(GetPeriodSummaryRequest) returns (GetPeriodSummaryResponse);
+}
+
+message GetPeriodSummaryRequest {
+  lifeos.common.v1.RequestMetadata metadata = 1;
```

Buf (1.72.0)

[error] cannot find `lifeos.common.v1.RequestMetadata` in this scope

(COMPILE)